mantle/platform/qemu.go

```diff
@@ -1394,9 +1394,20 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 	}
 	argv = append(argv, "-smp", fmt.Sprintf("%d", builder.Processors))
 
+	if builder.Firmware == "" {
+		switch coreosarch.CurrentRpmArch() {
+		case "x86_64", "aarch64":
+			builder.Firmware = "uefi"
+		}
+	}
+
 	switch builder.Firmware {
-	case "bios":
+	case "":
```

Comment:
So "" now means "the default, except on x86_64"?

Let's stick with the structure of the original code and always require a non-empty value here.  We can define a new name e.g. `platform` for ppc64le and s390x, assert that we only see it on those arches, and set it in `syncOptionsImpl`.